extensions/panache/hibernate-orm-panache-kotlin/runtime/src/test/kotlin/io/quarkus/hibernate/orm/panache/kotlin/runtime/TestAnalogs.kt

```diff
@@ -0,0 +1,90 @@
+package io.quarkus.hibernate.orm.panache.kotlin.runtime
+
+
+import io.quarkus.hibernate.orm.panache.kotlin.PanacheCompanion
+import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntityBase
+import io.quarkus.hibernate.orm.panache.kotlin.PanacheQuery
+import io.quarkus.hibernate.orm.panache.kotlin.PanacheRepositoryBase
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import java.lang.reflect.Modifier
+
+class TestAnalogs {
+    @Test
+    fun testJpaOperations() {
+        compare(map(io.quarkus.hibernate.orm.panache.runtime.JpaOperations::class.java), map(JpaOperations::class.java))
+    }
+
+    @Test
+    fun testPanacheQuery() {
+        compare(map(io.quarkus.hibernate.orm.panache.PanacheQuery::class.java), map(PanacheQuery::class.java))
+    }
+
+    @Test
+    fun testPanacheEntityBase() {
+        compare(map(io.quarkus.hibernate.orm.panache.PanacheEntityBase::class.java, true), map(PanacheCompanion::class.java),
+                listOf("findByIdOptional(arg0:Object, arg1:LockModeType)",
+                        "findByIdOptional(arg0:Object)",
+                        "deleteById(arg0:Object)"))
+        compare(map(io.quarkus.hibernate.orm.panache.PanacheEntityBase::class.java), map(PanacheEntityBase::class.java))
+    }
+
+    @Test
+    fun testPanacheRepositoryBase() {
+        compare(map(io.quarkus.hibernate.orm.panache.PanacheRepositoryBase::class.java), map(PanacheRepositoryBase::class.java))
+    }
+
+/*
+    @Test
+    fun testPanacheEntity() {
```

Comment:
Is this test disabled for a specific reason?

Reply:
Only that I'd forgotten I'd disabled it because it was leading to so much debug output at the time.  Reenabled, though.

Reply:
Mmm, it still appears disabled here. Commented out.